source/user-manual/capabilities/policy-monitoring/configuration_assessment/use-case.rst

```diff
@@ -0,0 +1,123 @@
+.. Copyright (C) 2019 Wazuh, Inc.
+
+Use case: Getting an alert when a check changes its result value
+================================================================
+
+To configure the execution of the *configuration assessment* module with a policy file, it is necessary to set up the following section:
+
+.. code-block:: none
+
+    <configuration_assessment>
+      <enabled>no</enabled>
+      <scan_on_start>yes</scan_on_start>
+      <interval>1m</interval>
+      <skip_nfs>yes</skip_nfs>
+      <profile>cis_debian_linux_rcl.yml</profile>
+    </configuration_assessment>
+
+The profile field is the policy file desired to be executed. The complete path for it is by default */var/ossec/etc/rootcheck* on a Linux manager, */var/ossec/etc/rootcheck/shared* on a Linux agent environment and *C:\\Program files (x86)\\ossec-agent\\shared* when using Windows.
+
+In this case, we have set a Debian Linux check file, but you can choose any other that matches your operating system.
```

Comment:
we -> In this case, a Debian Linux check file has been set.
you -> users
your -> their